_pages/index.md

```diff
@@ -3,33 +3,31 @@ title: Introduction
 permalink: /
 ---
 
-The goal of this site is to provide 18F staff with the information they need to launch software. It is a mix of requirements and best practices for projects at 18F, which may or may not be applicable for other teams inside or outside of government.
-
----
+The goal of this site is to provide 18F staff with the information they need to launch software. It's a mix of requirements and best practices for projects at 18F, which may or may not be applicable for other teams inside or outside of government.
 
 * **This list is:**
     * A general guide to review early and often
-    * _Especially_ important when you are starting to consider a future project launch or feature release
+    * _Especially_ important when you're starting to consider a future project launch or feature release
 * **This list is not:**
     * A last-minute pre-launch checklist, despite the "Before You Ship" title
 
-### Compliance
+## Compliance
```

Comment:
Mind changing these back? The page title is an `<h2>`, so we want these to be lower.